metadata.json

```diff
@@ -10,10 +10,10 @@
   "issues_url": "https://github.com/ghoneycutt/puppet-module-facter/issues",
   "description": "Manage Facter and optionally ensure facts.d directory.",
   "requirements": [
-    {"name":"puppet","version_requirement":">= 3.0.0 < 7.0.0"}
+    {"name":"puppet","version_requirement":">= 4.7.0 < 7.0.0"}
```

Comment:
This could be 5.8.. I certainly don't care about puppet 4 and don't want to bother testing against it.